metaviews/build.gradle

```diff
@@ -17,10 +28,19 @@ repositories {
 group 'com.mirego.trikot'
 
 kotlin {
+    ios {
+        binaries {
+            framework()
+        }
+    }
+    android {
+        publishAllLibraryVariants()
+    }
+
     targets {
         fromPreset(presets.jvm, 'jvm')
         fromPreset(presets.js, 'js')
-
+        fromPreset(presets.android, 'android')
```

Comment:
Is it ok to have both "jvm" and "android" as targets? The implementation of this new  service on jvm targets does nothing. So if a project is using jvm as its android target (which is already the case in some internal projects), it will get a version of the class that does nothing. 
Is it the responsability of the the client project to make sure it uses the right target? Maybe actually. But what is the point of a "jvm, non-android" target? For pure java applications? Is it really a case we want to support in the future?

Reply:
It is the responsability of the client projet to use the right target.